src/Digits.php

```diff
@@ -4,45 +4,36 @@
 
 namespace Laminas\Filter;
 
-use Laminas\Stdlib\StringUtils;
```

Comment:
Is `laminas/laminas-stdlib` still used a lot, after this change?

Reply:
Yes - it's still well used, but we may be able to drop it once we've got through all the filters :)

Reply:
Correct, for example `Laminas\Stdlib\ArrayUtils::iteratorToArray`.